tests/test_features.py

```diff
@@ -480,8 +478,10 @@ class Envelope(betterproto.Message):
     assert all([isinstance(item, datetime) for item in msg.timestamps])
 
 
-def test_enum_service_argument__expected_default_value():
-    from tests.output_betterproto.service.service import ThingType, TestStub
+def test_service_argument__expected_parameter():
```

Comment:
I completely changed what this test is about but kept it to a similar kind of test, asserting what we can see via inspect.

Reply:
Why exactly have you removed this?

Reply:
In my mind I was thinking an rpc method wouldn't take an enum, but now I'm wondering if perhaps enum is allowed as the param in a proto file. Assuming that enums are allowed as the parameter to a proto, do we want to provide default value for enum in the function signature? With this change atm there's no longer any defaults in the RPC method signatures.

After some googling though, I don't think direct enums are a valid rpc parameter they seem more like scalars than messages.

Lmk if you know the answer to this, or I'll test it out myself.

Reply:
No you're completely correct my bad